pkg/activator/config/store.go

```diff
@@ -51,15 +53,23 @@ func NewStore(logger configmap.Logger, onAfterStore ...func(name string, value i
 	// Append an update function to run after a ConfigMap has updated to update the
 	// current state of the Config.
 	onAfterStore = append(onAfterStore, func(_ string, _ interface{}) {
-		s.current.Store(&Config{
-			Tracing: s.UntypedLoad(tracingconfig.ConfigName).(*tracingconfig.Config).DeepCopy(),
-		})
+		c := &Config{}
```

Comment:
We need this (watching config-network via activator) in the future but current implementation still needs to restart activator for internal encryption on/off.

If we add this, I think we should add a comment with the issue link  https://github.com/knative/serving/issues/13754